Modules/Core/Common/include/itkByteSwapper.hxx

```diff
@@ -39,202 +39,170 @@ namespace itk
 //------Big Endian methods----------------------------------------------
 
 // Use different swap methods based on type
-#ifdef CMAKE_WORDS_BIGENDIAN
-template <typename T>
-void
-ByteSwapper<T>::SwapFromSystemToBigEndian(T *)
-{}
-#else
+
 template <typename T>
 void
 ByteSwapper<T>::SwapFromSystemToBigEndian(T * p)
```

Comment:
We may need to add `[[maybe_unused]]` here now!

Reply:
Indeed, the original PR did cause warnings, at https://open.cdash.org/viewBuildError.php?type=1&buildid=9892277, like:

> itkByteSwapper.hxx:153:55: warning: parameter 'p' set but not used [-Wunused-but-set-parameter]

These should all be prevented now, with [the force-push that I just did](https://github.com/InsightSoftwareConsortium/ITK/compare/3d4f5251cb6ad032a21b4be00ad551230233c3ab..a38ef793f4e7dbdd964f7d57c1bfc1e571e9670d).